### ISSUE TITLE
Hide MARK:-

### ISSUE THREAD

**Issue by drekka** [realm/jazzy#857]
Is it possible to stop Jazzy from adding any `MARK:-` as subtitles in the generated html?

I used them quite extensively to help organise my methods, but I don't want them in the generated documentation.

**Comment by johnfairh** [comment]
Nope, it's a feature that they're carried through.  You could try hiding them via CSS - the `section-name` class is the one for those subtitles and `task-name-container` for the div around that.

**Comment by reitzig** [comment]
A way to hide obsolete MARKs would be great. Example:

```
class Foo {
	// MARK: Doing Stuff

	/// ...
	public func foo() { ... }

	// MARK: Helpers

	/// ...
	fileprivate func bar() { ... }
}
```

Now, both MARKs would show even though the second one quite obviously marks nothing.

Proposal: drop MARKs which are not followed by an actual documented symbol, but by EOF or another MARK.

Maybe this can be worked around with using CSS as well, but it sure is not immediate.

**Comment by reitzig** [comment]
Never mind, apparently Jazzy already does what I propose. :'D

**Comment by drekka** [comment]
I managed to hide the marks by adding my own css file and hiding them. Ugly but worked. I used this in my CSS:

    div.task-name-container {
        display:none;
    }
    .main-content ul {
        margin-bottom:0;
    } 

And then added this to my jazzy command:

    --head '<link rel="stylesheet" type="text/css" href="files/My.css" /><link rel="stylesheet" type="text/css" href="../files/My.css" />'

Note that I had to add it twice using two different URLs. This solves the issue where CSS files are reference both in the root of the documentation and in subdirectories. If there is a better way of handling this, I'd like to know.

**Comment by reitzig** [comment]
@drekka Thanks for sharing! I already hack the CSS, but I for one don't want to hide *all* marks *everywhere*. :/

FWIW, I maintain my custom CSS in its own file, and do this after Jazzy has run:

	cat doc/custom.css >> build/doc/css/jazzy.css
